LanguageFeatures/Extension-types/dynamic_semantics_member_invocation_A03_t07.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Consider an invocation of the extension type member m on the
+/// receiver expression e according to the extension type declaration V with
+/// actual type arguments T1, ..., Ts. If the invocation includes an actual
+/// argument part (possibly including some actual type arguments) then call it
+/// args. If the invocation omits args, but includes a list of actual type
+/// arguments then call them typeArgs. Assume that V declares the type variables
+/// X1, ..., Xs
+/// ...
+/// Let Dm be the unique declaration named m that V has.
+///
+/// Evaluation of this invocation proceeds by evaluating e to an object o.
+/// ...
+/// Otherwise, the following is known: args is included, and Dm is a method. The
```

Comment:
This paragraph describes method invocations, not property extractions (also known as tear-offs).

I think this paragraph is more relevant:

> Otherwise, if `args` is omitted and _Dm_ is a method, ...

Reply:
Same issue with other tests.

Reply:
Tests renamed to `dynamic_semantics_member_invocation_A02_*` because these tests chek this statement